tools/cli/commands/changelog.js

```diff
@@ -743,6 +743,17 @@ async function promptChangelog( argv, needChangelog, types ) {
 		},
 	} );
 
+	let userFacing = false;
+	// Ask if this is a user facing change if we detect Jetpack.
+	if ( argv.project === 'plugins/jetpack' ) {
```

Comment:
I'm pretty sure this will only trigger for `jetpack changelog add plugins/jetpack`. We should do something like this to handle when `jetpack changelog add` detects Jetpack too.
```suggestion
	if ( needChangelog.includes( 'plugins/jetpack' ) ) {
```

Reply:
Oh duh, nice catch.